services/server/src/server/apiv1/verification/private/stateless/customReplaceMethods.ts

```diff
@@ -0,0 +1,164 @@
+import { VerificationExport } from "@ethereum-sourcify/lib-sourcify";
+import { PoolClient } from "pg";
+import {
+  bytesFromString,
+  getDatabaseColumnsFromVerification,
+} from "../../../../services/utils/database-util";
+
+export type CustomReplaceMethod = (
+  poolClient: PoolClient,
+  verification: VerificationExport,
+) => Promise<void>;
+
+export const replaceCreationInformation: CustomReplaceMethod = async (
+  poolClient: PoolClient,
+  verification: VerificationExport,
+) => {
+  if (
+    verification.status.creationMatch !== "perfect" &&
+    verification.status.creationMatch !== "partial"
+  ) {
+    throw new Error(
+      "Creation match is null, cannot replace creation information",
+    );
+  }
```

Comment:
With my other comment, I meant to check if the `VerificationExport` contains creation bytecode. I think for the contracts affected by #1990, it is still possible that there is no creation match.